src/utilities/profiles/render.ts

```diff
@@ -0,0 +1,51 @@
+import { ProfileData } from "../../types/Interfaces";
+import { EmbedBuilder } from "discord.js";
+import Badges from "./badges";
+import badges from "./badges";
+import Query from "../../routes/Query";
+
+export default async function render(profile: ProfileData) {
+  // Profile data destructure
+  let display_name = profile.display_name;
+  let badge_flags = profile.badge_flags;
+  let user_id = profile.user_id;
+  let bio = profile.bio;
+  let display_picture = profile.display_picture ?? process.env.CLIENT_PFP;
+  let hits = profile.hits;
+  let likes = profile.likes;
+  let id = profile.id;
+  let muted = profile.muted;
+
+  // Preliminary data parsing
+  let embedColor = "Blurple";
```

Comment:
Prefer Hello World! theme.
```suggestion
  let embedColor = "#5856d6";
```